wire-ios-data-model/WireDataModel.xcodeproj/project.pbxproj

```diff
@@ -57,7 +56,6 @@
 		068D610324629AB900A110A2 /* ZMBaseManagedObjectTest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 068D610124629AA300A110A2 /* ZMBaseManagedObjectTest.swift */; };
 		068DCC5729BB816300F7E4F1 /* ZMOTRMessage+FailedToSendReason.swift in Sources */ = {isa = PBXBuildFile; fileRef = 068DCC5629BB816300F7E4F1 /* ZMOTRMessage+FailedToSendReason.swift */; };
 		068DCC5B29BFEC4E00F7E4F1 /* ZMMessageTests+ExpirationReasonCode.swift in Sources */ = {isa = PBXBuildFile; fileRef = 068DCC5A29BFEC4E00F7E4F1 /* ZMMessageTests+ExpirationReasonCode.swift */; };
-		069BCC4E2B2CA47700DF4EC2 /* store2-112-0.wiredatabase in Resources */ = {isa = PBXBuildFile; fileRef = 069BCC4D2B2CA47700DF4EC2 /* store2-112-0.wiredatabase */; };
```

Comment:
I don't think we should remove it. Same for `store2-113-0.wiredatabase`

Reply:
The file reference isn't actually removed, these seem to be duplicate entries from conflict solving.